layouts/_default/single.html

```diff
@@ -35,7 +35,10 @@ <h1>{{ if .Draft }}DRAFT: {{end}}{{ .Title }}</h1>
       </div>
 
       <div class="post-content">
-        {{ .Content }}
+        {{ if or .Params.enable_toc (and (eq .Params.enable_toc nil) .Site.Params.enable_toc) -}}
```

Comment:
Can you change the parameter to just `toc`?

Reply:
And can you also document this somewhere in the README? Preferably give a shout to it in the `config` example.

Reply:
Parameter name changed.

I document it in README, not duplicate in `exampleSite/config.toml` because the latter is cleaner and I guess people will prefer README first.  Feel free to ask me update `config.toml` too.